mittab/apps/tab/migrations/0011_auto_20191010_2104.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.1.5 on 2019-10-10 21:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tab', '0010_auto_20191008_2358'),
+    ]
+
+    operations = [
```

Comment:
Is this a git issue? Any reason this change is on this PR?

Reply:
Fixed!

Reply:
Ah I see -- is the issue that the migration wasn't added in the not-on-multiple-teams PR, but django application logic still checks for uniqueness without the migration, so that PR worked even though the migration wasn't added? And now when you try to run `migrate` it asked you to `makemigrations` first?

Reply:
^ If so, just open another PR to add this migration

Reply:
I'll check this out.

Reply:
#261 solves